File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;
import entity.Result;

public interface BrandService {
	
	
	List<TbBrand> findAll();
	
	PageResult findPage(int page,int size);
	
	void add(TbBrand brand);

	void update(TbBrand brand);

	TbBrand findOne(Long id);

	Result delete(Long[] ids);

	PageResult search(TbBrand searchEntity, int page, int size);
	
	List<Map> selectOptionList(); 

}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function( $http){
		//查询所有商标
	 	this.findAll = function() {
			return	$http.get('../brand//findAll.do')
		};
		
	 	//查询所有品牌并分页
		this.findPage = function(page, size) {
			return $http.get('../brand/findPage.do?page=' + page + '&size='+ size)
		};
		
		//添加品牌方法
		this.add=function(entity){
			return $http.post("../brand/add.do",entity);
		}
		
		//修改品牌方法
		this.update=function(entity){
			return $http.post("../brand/update.do",entity);
		}
		
		//根据id查询品牌
		this.findOne=function(id){
			return $http.get("../brand/findOne.do?id="+id);
		}
		
		//删除品牌
		this.dele=function(selectIds){
			return $http.get("../brand/delete.do?ids="+selectIds);
		}
		
		//条件查询
		this.search=function(page,size,searchEntity){
			return $http.post("../brand/search.do?page=" + page + "&size="+ size,searchEntity)
		}
		//返回所有下拉列表商标
		this.selectOptionList=function(){
			return $http.get("../brand/selectOptionList.do")
		}
		
	});